packages/service-user/src/lib/account/queries/get-participants.ts

```diff
@@ -0,0 +1,19 @@
+import { sql } from '@blms/database';
+
+export const getParticipantsForEvent = (eventId: string) => {
```

Comment:
We could use an indiscriminate query instead (getting back all the events and not just the one with the corresponding id) -> would prevent making many small queries